packages/jest-get-type/src/index.ts

```diff
@@ -57,7 +57,7 @@ export function getType(value: unknown): ValueType {
     return 'symbol';
   }
 
-  throw new Error(`value of unknown type: ${value}`);
+  throw new Error(`value of unknown type: ${typeof value} (${value})`);
```

Comment:
`typeof null === 'object'`, but I guess that's already handled above. Any other such edge cases?

Reply:
Not that I am aware of.